README.md

```diff
@@ -1,6 +1,11 @@
 # Windows Azure Active Directory Authentication Library (ADAL) for Node.js
 The ADAL for node.js library makes it easy for node.js applications to authenticate to AAD in order to access AAD protected web resources.  It supports 3 authentication modes shown in the quickstart code below.
 
+## Versions
+Current version - 0.1.22  
+Minimum recommended version - 0.1.22  
```

Comment:
adal node is still under preview version, we should always recommend the devloper use the latest version. @addev-ashish correct me if i'm wrong.